pkg/core/resources/apis/meshtrust/api/v1alpha1/meshtrust.go

```diff
@@ -1,7 +1,7 @@
 // +kubebuilder:object:generate=true
 package v1alpha1
 
-// MeshTrust
+// MeshTrust defines trusted Certificate Authority (CA) bundles for a trust domain in the mesh. It establishes trust relationships for service-to-service mTLS authentication by specifying which CA certificates are trusted to verify service identities, supporting PEM-encoded CA bundles and enabling secure cross-service communication within the trust domain.
```

Comment:
Please use the suggestion feature on the go object comment that way I can add the commit and simply regenerate
```suggestion
// MeshTrust defines trusted Certificate Authority (CA) bundles for a trust domain in the mesh. It establishes trust relationships for service-to-service mTLS authentication by specifying which CA certificates are trusted to verify service identities, supporting PEM-encoded CA bundles and enabling secure cross-service communication within the trust domain.
```